frappe/public/js/frappe/list/list_view.js

```diff
@@ -268,7 +268,7 @@ frappe.views.ListView = class ListView extends frappe.views.BaseList {
 		if (frappe.route_options) {
 			// Priority 1: route filters
 			this.filters = this.parse_filters_from_route_options();
-		} else if (this.view_user_settings.filters) {
+		} else if (this.view_user_settings.filters && this.view_user_settings.filters) {
```

Comment:
redundant check?

Reply:
Sorry it should be this.view_user_settings.filters.length